requirements/dev.txt

```diff
@@ -5,92 +5,85 @@
 #    make upgrade
 #
 -e git+https://github.com/edx/acid-block.git@98aecba94ecbfa934e2d00262741c0ea9f557fc9#egg=acid-xblock
-appdirs==1.4.3            # via fs
-astroid==1.6.6            # via pylint, pylint-celery
-atomicwrites==1.3.0       # via pytest
+appdirs==1.4.3            # via fs, virtualenv
+astroid==2.3.3            # via pylint, pylint-celery
```

Comment:
@kaizoku, this breaks the CI checks [here](https://travis-ci.org/edx/xblock-sdk/jobs/654252329#L415).